app/models/scc_account.rb

```diff
@@ -278,4 +285,18 @@ def invalidate_subscription_status(items_to_invalidate, save_record:)
     end
     items_to_invalidate
   end
+
+  def self.download_policy_selection_values
+    names = { 'on_demand' => _('On Demand'), 'immediate' => _('Immediate') }
+    ::Katello::RootRepository::DOWNLOAD_POLICIES.map do |p|
+      names.include?(p) ? [names[p], p] : [p, p]
```

Comment:
I was reading through this and a more efficient way is to use [fetch](https://ruby-doc.org/core-3.0.1/Array.html#method-i-fetch) with a default value:
```suggestion
      [names.fetch(p, p), p]
```